contracts/token/ERC20/extensions/ERC20FlashMint.sol

```diff
@@ -19,6 +19,21 @@ import "../ERC20.sol";
 abstract contract ERC20FlashMint is ERC20, IERC3156FlashLender {
     bytes32 private constant _RETURN_VALUE = keccak256("ERC3156FlashBorrower.onFlashLoan");
 
+    /**
+     * @dev The loan token is not valid.
+     */
+    error ERC3156InvalidToken(address token);
```

Comment:
The EIP says:
> The flashFee function MUST return the fee charged for a loan of amount token. If the token is not supported flashFee MUST revert.

So to match the EIP wording I would do 
```suggestion
    error ERC3156UnsupportedToken(address token);
```

Reply:
Agree with this, thank you for the catch!